docs/60_initialize_vendors.md

```diff
@@ -0,0 +1,448 @@
+# Initialize Vendor Accounts
+
+The DANDI ecosystem relies on vendor services to operate.  So first you will need to set up accounts with the following vendors:
+
+• **Heroku**
+
+• **AWS**
+
+• **GitHub**
+
+• **Terraform Cloud**
+
+• **Netlify**
+
+• **Sentry**
+
+• **PyPI**
+
+• **Datalad (TBD)**
+
+• **git-annex (TBD)**
```

Comment:
Those are just tools -- no account needed and overall they just rely on above services (GitHub) and git configuration to run.  

But for completeness -- we do need a host (in DANDI case it is `drogon` server) with an account under which to run all those additional "cron jobs", hence

```suggestion

In addition a host (local server or an instance in cloud) is needed to run additional services employing DataLad and git-annex.
```

Reply:
are cron-jobs part of the infrastructure? or should be considered essential?

Reply:
We do not have definition of "infrastructure" to define the boundary. 

if we consider https://github.com/dandisets etc as part of infrastructure, then yes.

as "essential" -- likely not as long as not integrated within dandiarchive.org web UI.